omnigibson/prims/entity_prim.py

```diff
@@ -306,8 +274,8 @@ def _update_joint_limits(self):
             else:
                 assert not self.initialized, \
                     "Cannot update joint limits for a non-uniformly scaled object when already initialized."
-                for link_name, link in self.links.items():
-                    if joint.parent_name == link_name:
+                for link in self.links.values():
+                    if joint.body0 == link.prim_path:
```

Comment:
parent_name was the non-fully-qualified name received from dynamic control, this is no longer available in the new API, so that parent/child name API is removed from the OG classes too.